### PR TITLE
staking form ui refinement

### PR DESCRIPTION
**Pull request by supertong**
Demo:

image

image

### CODE REVIEW

File: src/app/components/Staking/Form/StakingFee.tsx [hunk 1]
```diff
@@ -48,31 +50,58 @@ export const StakingFee: React.FC<StakingFeeProps> = ({
...
-                  {satoshiToBtc(stakingFeeSat)} {coinName}
-                </strong>
-              </p>
+      <div className="flex flex-col gap-6">
+        {mempoolFeeRates ? (
+          <>
+            <div className="flex flex-row items-center justify-between text-primary-dark">
+              <Text variant="body1">Network Fee Rate</Text>
+              <div className="flex flex-row gap-2 items-center">
+                <Text variant="body1">25 Sats/vB</Text>
```
Comment (jeremy-babylonlabs) [inline src/app/components/Staking/Form/StakingFee.tsx:59]: I think we can use the `defaultFeeRate` here?

**Review by jeremy-babylonlabs — APPROVED** [review]
nice, lgtm!

File: src/app/components/Staking/Form/StakingAmount.tsx [hunk 2]
```diff
@@ -118,24 +121,61 @@ export const StakingAmount: React.FC<StakingAmountProps> = ({
...
+      <div className={"flex flex-col gap-6"}>
+        <div className="flex flex-col">
+          <div
+            className={twJoin(
+              "flex flex-col gap-2 border rounded border-primary-light p-3",
+              error ? "border-error-main" : "",
+            )}
+          >
+            <div className="flex flex-row items-center">
+              <input
```
Comment (jeremy-babylonlabs) [inline src/app/components/Staking/Form/StakingAmount.tsx:137]: nit: I think input field should allow numbers only

**Review by totraev — APPROVED** [review]
LGTM

**Comment by totraev** [comment]
@supertong great job! Can we align error message by left side?

**Pull request merged**